src/assets/scss/abstract/_variables.scss

```diff
@@ -38,3 +38,8 @@ $padding: 1rem;
 $padding-md: 0.75rem;
 $padding-sm: 0.5rem;
 $padding-xs: 0.25rem;
+
+$sm: 576px;
```

Comment:
@daneyul looks was left over too, but going to merge as is for now.  we can cleanup later.
maybe we can use these too and combine with layout breakpoints